docs/ops/doc/SearchingForOps.md

```diff
@@ -96,35 +109,33 @@ Names:
 	> filter.variance
 ```
 
-## Signatures for Op Names
+## Querying Op Signatures
 
 Finally, you can use `OpEnvironment.help()` on any Op name to see the list of signatures:
 
-```groovy
-import org.scijava.ops.api.OpEnvironment
-ops = OpEnvironment.build()
-
+```
 print(ops.help("filter.gauss"))
 ```
 
-```
+```text
 filter.gauss:
 	- (input, sigmas, @CONTAINER container1) -> None
 	- (input, sigmas, outOfBounds = null, @CONTAINER container1) -> None
```

Comment:
Not part of this PR I know, but this output mixes `null`s and `None`s... we should make that more consistent.